tests/linux_minidump_writer_failspot.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg(any(target_os = "linux", target_os = "android"))]
```

Comment:
This test had to be put in its own file because current `failspot` tests cannot be run concurrently with non-`failspot` tests. I am hoping to change that eventually, but right now that's how it is...